src/extension.ts

```diff
@@ -1,180 +1,38 @@
 import * as vscode from "vscode";
-import * as path from "path";
-import { pyRunApp, rRunApp, pyDebugApp, onDidStartDebugSession } from "./run";
-import {
-  shinyliveCreateFromActiveEditor,
-  shinyliveSaveAppFromUrl,
-  shinyliveCreateFromExplorer,
-} from "./shinylive";
 
-export function activate(context: vscode.ExtensionContext) {
-  context.subscriptions.push(
-    vscode.commands.registerCommand("shiny.python.runApp", pyRunApp),
-    vscode.commands.registerCommand("shiny.python.debugApp", pyDebugApp),
-    vscode.commands.registerCommand("shiny.r.runApp", rRunApp),
-    vscode.commands.registerCommand(
-      "shiny.shinylive.createFromActiveEditor",
-      shinyliveCreateFromActiveEditor
-    ),
-    vscode.commands.registerCommand(
-      "shiny.shinylive.saveAppFromUrl",
-      shinyliveSaveAppFromUrl
-    ),
-    vscode.commands.registerCommand(
-      "shiny.shinylive.createFromExplorer",
-      shinyliveCreateFromExplorer
-    )
+export async function activate(
```

Comment:
Either don't do this at all, or, unconditionally offer to uninstall yourself. (And remove that offer from `Posit.shiny`)

Reply:
Would it be crazy to unconditionally uninstall, without even asking?

Reply:
I went with the unconditional uninstall. I thought about including an informational message telling the user about the action but decided against it. Let me know if you'd like to add that back in.